engines/qdengine/qdengine.cpp

```diff
@@ -319,6 +319,8 @@ void QDEngineEngine::syncSoundSettings() {
 
 	sndDispatcher::get_dispatcher()->syncSoundSettings();
 	mpegPlayer::instance().syncMusicSettings();
+
+	ConfMan.flushToDisk();
```

Comment:
This is wrong place. It is breaking the whole idea and design.

Reply:
Should be fixed now. @sev-